wrangler.toml

```
@@ -0,0 +1,11 @@
+name = "firefox-cro-redirector"
+type = "javascript"
+workers_dev = false
+
+[env.staging]
+name = "firefox-cro-redirector"
```

Can we rename this more explicitly to stage?

```suggestion
name = "firefox-cro-redirector-stage"
```

---

ok with me.  We'll need to delete the existing stage worker.  Wasn't sure if that had implications for the CRO team, so wanted to keep it as is. (Do you know, is it safe to delete/recreate?).

---

> (Do you know, is it safe to delete/recreate?).

I think once we have the pipeline running the existing stage worked should be fine to delete 👍

---

Planning on doing the rename in a different commit, so we can sync on timing for it.